apps/examples/ble_rmc/ble_rmc_main.c

```diff
@@ -103,15 +103,15 @@ static void ble_device_scanned_cb_for_connect(ble_scanned_device *scanned_device
 	}
 }
 
-static void ble_device_disconnected_cb(ble_client_ctx *ctx)
+static void ble_device_disconnected_cb(ble_client_ctx *ctx, int reason)
```

Comment:
How about get structure type not integer(integer). Because requirements can be changed later, then you need to add more field to the callback. 
For example before there is no requirement to get AP Mac address in wi-fi manager. however as new application appears, new requirement for a driver comes. Please think about it.

Reply:
Thanks for your advice.
Most information related to connection is included in ble_client_ctx. 
So I think a new struct for reason would not be needed.